src/storage/src/region/writer.rs

```diff
@@ -336,6 +337,7 @@ impl RegionWriter {
             files
         );
 
+        drop_ctx.manifest.manifest_store().delete_all().await?;
```

Comment:
What's more, we need to handle the `Delete` action in `recover_from_manifest()`. If we find out the delete action, we can rerun `manifest.delete_all()` and then return `None` to indicate there is no such region.
https://github.com/GreptimeTeam/greptimedb/blob/559d1f73a2819b775b9c8757cff08522ec216f14/src/storage/src/region.rs#L459-L490

Reply:
Got it!